crunchy-xml-decoder/ultimate.py

```diff
@@ -241,22 +242,26 @@ def ultimate(page_url, seasonnum, epnum):
     vid_id = xmlconfig.find('media_id').string
 
     # ----------
-
-    try:
-        host = xmlconfig.find('host').string
-    except AttributeError:
+    host = xmlconfig.find('host')
+    if host is not None:
```

Comment:
I didn't want to worry about what qualifies as truth for the query result. So it might be cleaner to just check it for truthness directly, but the `is None` certainly expresses exactly the absence of the node.